openslides_backend/action/actions/user/create.py

```diff
@@ -60,8 +58,17 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             or instance.get("last_name")
         ):
             raise ActionException("Need username or first_name or last_name")
-        if OrganisationManagementLevel(instance.get("organisation_management_level", "no_right")) < OrganisationManagementLevel.CAN_MANAGE_USERS and len(instance.get("group_$_ids", [])) == 0 and len(instance.get("committee_ids", [])) == 0:
-            raise ActionException("To create a user you need to add him to a permission-group, add him to a committee or give him an Organisation Management Level of at least 'can manage users'.")
+        if (
+            OrganisationManagementLevel(
+                instance.get("organisation_management_level", "")
```

Comment:
You should be able to remove the default value, OML can handler `None`. If mypy complains, then you could try to change the definition here: https://github.com/OpenSlides/openslides-backend/blob/26e286c3216ae4cb6e4982373c1db56d413fcfb4/openslides_backend/permissions/management_levels.py#L9 to `value: Any` - I favor shorter code over mypy appeasement ;)

Reply:
but I just noticed that below for CML mypy doesn't complain, so maybe it just works without the change

Reply:
You are  right, works without further changes :-)

Reply:
Exactly, I wanted to write this as a suggestion, but somehow this was deactived this morning...

Reply:
Ah, it wasn't pushed yet ;)